### DISCUSSION TITLE
令牌功能出现了问题

### DISCUSSION THREAD

**Discussion by Utopia1916** [BeyondDimension/SteamTools#2740] [General]
确认交易功能异常，无法获取交易信息，提示获取交易数据失败，未加速steam，注销登陆后尝试，无法登陆，同样提示未加速steam，目前尝试了重启应用，重启电脑，更换多个VPN，更换多个节点，更换多种代理方式，仍无法解决，更换手机APP，同样的结果，我最后一次使用确认交易功能是在这次steam更新UI前，今天是更新后第一次使用，软件版本，使用检查更新功能提示已是最新版，2.8.4 for windows desktop bridege(x64)，我不知道是只有我出现了这个问题还是其他人同样出现了这个问题，我也发送了邮件但是没有很详细的描述，希望开发者尽快查看并解决，谢谢

**Comment by rmbadmin** [comment]
因为steam昨天下掉了旧的接口，目前只能等更新新版本解决

**Reply by Utopia1916** [reply]
？？(oˍO)早点休息吧，之前谈恋爱的时候我女朋友回我都没你快

**Reply by Utopia1916** [reply]
秋梨膏，什么时候更新啊，马上夏促了，major胶囊也打折了，软妹币我没有你不行啊😭😭😭😭😭，我那么多账号/(ㄒoㄒ)/~~

**Comment by GLintliang** [comment]
我也是相同的问题

**Reply by Utopia1916** [reply]
看到别人有找到浏览器脚本来临时代替的办法，但说实话我不太敢用
